src/styles/blocks/nav.scss

```
@@ -37,5 +22,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

Необходимо явно указывать свойства, которые будут анимированы, иначе используется transition: all
transition: all достаточно требователен к ресурсам браузера и на старых компьютерах может ухудшать пользовательский опыт
```suggestion
    transition-duration: 300ms;
    transition-property: transform;
```

либо transition: transform 300ms;